dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -13,57 +13,80 @@
  */
 package com.dynatrace.opentelemetry.metric;
 
+import com.dynatrace.metric.util.Dimension;
+import com.dynatrace.metric.util.DimensionList;
+import com.dynatrace.metric.util.MetricBuilderFactory;
 import com.google.common.annotations.VisibleForTesting;
-import io.opentelemetry.api.metrics.common.Labels;
+import com.google.common.base.Charsets;
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+import com.google.common.io.CharStreams;
 import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
 import io.opentelemetry.sdk.metrics.data.MetricData;
 import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
-import java.util.AbstractMap;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import javax.annotation.Nonnull;
 
 /** Export metric to Dynatrace. */
 public final class DynatraceMetricExporter implements MetricExporter {
   private final URL url;
   private final String apiToken;
+  private final Serializer serializer;
 
   private static final Logger logger = Logger.getLogger(DynatraceMetricExporter.class.getName());
+  private static final List<Dimension> staticDimensions =
+      new ArrayList<Dimension>() {
+        {
+          add(Dimension.create("dt.metrics.source", "opentelemetry"));
+        }
+      };
+
+  private static final Pattern EXTRACT_LINES_OK = Pattern.compile("\"linesOk\":\\s?(\\d+)");
+  private static final Pattern EXTRACT_LINES_INVALID =
+      Pattern.compile("\"linesInvalid\":\\s?(\\d+)");
+  private static final Pattern IS_NULL_ERROR_RESPONSE = Pattern.compile("\"error\":\\s?null");
 
   private DynatraceMetricExporter(
       URL url,
       String apiToken,
       String prefix,
-      Labels defaultDimensions,
+      DimensionList defaultDimensions,
```

Comment:
Yes you are right. I am not really sure if we want to do it this way since `Labels` are  provided by OpenTelemetry, while we supply the `DimensionList` ourselves, in turn requiring users to include our Metrics library in their project in order to use the exporter. If we use `Labels` and transform them internally, that would not only not be a breaking change, it would also allow us to "hide" the library from users as a transitive dependency.

Reply:
Moved it back to `Labels` which also allows us to remove the dependency on `dynatrace-metric-utils` from the example project.

Reply:
Using otel types where possible I think is a good idea, and avoiding requiring users to know about and use our internal metric utils library is also good.